packages/grid-pro/src/vaadin-grid-pro-edit-column-mixin.js

```diff
@@ -267,8 +275,17 @@ export const GridProEditColumnMixin = (superClass) =>
       this._setEditorValue(editor, get(this.path, model.item));
       editor._grid = this._grid;
 
-      this._focusEditor(editor);
-      requestAnimationFrame(() => this._focusEditor(editor));
+      const editInitiatorKey = this._editInitiatorKey;
+      this._editInitiatorKey = undefined;
+      requestAnimationFrame(() => {
+        if (editInitiatorKey) {
```

Comment:
Previously it was possible to paste text into a cell, for example with <kbd>Command + V</kbd>. When I try this now I only get a `v` in the cell editor.

Reply:
Maybe a solution would be then to save the keyboard input into a temporary input while the editor hasn't rendered yet.

Reply:
This approach was abandoned. Currently, the Polymer editor is focused synchronously and the Lit editor is focused when it is ready.